.pre-commit-hooks.yaml

```diff
@@ -1,10 +1,10 @@
--   id: prettier
-    name: prettier
-    description: ''
-    entry: prettier --write --ignore-unknown
-    language: node
-    'types': [text]
-    args: []
-    require_serial: false
-    additional_dependencies: ["prettier@3.3.2"]
-    minimum_pre_commit_version: '0'
+- id: prettier
+  name: prettier
+  description: ""
+  entry: prettier --write --ignore-unknown
+  language: node
+  "types": [text]
+  args: [--cache-location, .prettier-cache]
```

Comment:
I think this should be in entry so it's not removed if args is overriden, also this part is generated see the github actions.

Reply:
I'll take a look at the action and update, thanks.

Wrt `args`, I put it in there rather than `entry` on the basis a user may be passing in the `--cache` and `--cache-location` as arguments because they are actually using it.  The bug only affects people that aren't using the cache, so seemed sensible to allow overriding?